lib/need_for_type/states/start.rb

```diff
@@ -10,9 +10,11 @@ def initialize(display_window)
     end
 
     def update
-      @display_window.render_menu(@option)
+      @display_window.render_init_game
```

Comment:
I think we should rename `render_init_game` function to `render_start`, to make it more clear.

And following the same pattern we should also rename `render_score` to `render_end`, so that every state has it's own rendering function.